### PR TITLE
[zh-cn] sync task files of task-15

### PR DESCRIPTION
**Pull request by SSmallMonster**
Part of `task-15`: https://github.com/kubernetes/website/issues/39983

### CODE REVIEW

**Comment by SSmallMonster** [comment]
> This behavior is similar to a spam. Please combine them into one PR. Otherwise, it may waste much time to reviewers and yours. Hope you can understand. #40159, #40160, #40161, #40162
> 
> image
> 
> /hold

Thanks for the reminder, a new PR includes https://github.com/kubernetes/website/pull/40159, https://github.com/kubernetes/website/pull/40160, https://github.com/kubernetes/website/pull/40161, https://github.com/kubernetes/website/pull/40162 will be raised later.

**Comment by howieyuen** [comment]
/hold cancel
/retitle [zh-cn] sync task files of task-15

File: content/zh-cn/docs/tasks/configmap-secret/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 title: "管理 Secrets"
-weight: 28
+weight: 60
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/_index.md:3]: The frontmatter in english can be copied and commented here.

File: content/zh-cn/docs/tasks/inject-data-application/_index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
 title: "给应用注入数据"
-weight: 30
+weight: 70
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/inject-data-application/_index.md:3]: The english frontmatter can be copied and commented here, to keep consistent with other pages.

File: content/zh-cn/docs/tasks/job/_index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
 title: "运行 Jobs"
-weight: 50
+weight: 90
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/_index.md:3]: The english frontmatter can be copied and commented here, to keep consistent with other pages.

**Review by windsonsea — COMMENTED** [review]
Pick a nit.

File: content/zh-cn/docs/tasks/configmap-secret/_index.md (hunk 1 above)
Comment (SSmallMonster) [reply content/zh-cn/docs/tasks/configmap-secret/_index.md:3]: updated

File: content/zh-cn/docs/tasks/job/_index.md (hunk 3 above)
Comment (SSmallMonster) [reply content/zh-cn/docs/tasks/job/_index.md:3]: updated

File: content/zh-cn/docs/tasks/configmap-secret/_index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
 title: "管理 Secrets"
```
[the remark this answers is not in the document]
Comment (SSmallMonster) [reply content/zh-cn/docs/tasks/configmap-secret/_index.md:2]: 好的，已经去掉复数更新了

**Pull request merged**